packages/editor-ui/src/plugins/i18n/docs/ADDENDUM.md

```diff
@@ -103,12 +103,99 @@ Currently only the keys `oauth.clientId` and `oauth.clientSecret` are supported
 
 ### Special cases
 
-`eventTriggerDescription` is a dynamic node property that is not part of node parameters. To translate it, set the `eventTriggerDescription` key at the root level of the `nodeView` property in the node translation file.
+`eventTriggerDescription` and `activationMessage` are dynamic node properties that are not part of node parameters. To translate them, set the key at the root level of the `nodeView` property in the node translation file.
+
+Webhook node:
 
 ```json
 {
 	"nodeView": {
-		"eventTriggerDescription": "🇩🇪 Waiting for you to call the Test URL"
+		"eventTriggerDescription": "🇩🇪 Waiting for you to call the Test URL",
+	}
+}
+```
+
+Cron node:
+
+```json
+{
+	"nodeView": {
+		"activationMessage": "🇩🇪 'Your cron trigger will now trigger executions on the schedule you have defined."
+	}
+}
+```
+
+## Best practices for UI development
+
+When creating a new component, create a new component key in `en.json` with the component name in camel case. Place the key in `en.json` in alphabetical order.
+
+In most cases, the new component is created in `/components`, e.g. `/components/About.vue`. But if the component is created in a nested dir, e.g. `/components/CredentialEdit/CredentialConfig.vue`, ensure this nesting is reflected in `en.json`.
+
+```json
+{
+	"credentialEdit": {
+		"credentialConfig": {
+			"accountConnected": "Account connected"
+		}
```

Comment:
is this really a good idea? would not this mean we have to rename keys every time we refactor. would prefer to keep keys as flat as possible

Reply:
In addition to the dir case, there are potentially more levels of nesting: dir > component > toast. Rephrasing a display string, renaming a component or dir, refactoring a component, they all require updating the translation file.

I don't disagree that this coupling is not ideal, but if we flatten the object, the translations file becomes less organized, and we lose the ability to translate the same keys in different ways when in different contexts. Enforcing nesting to avoid key clashes was the motivation for reworking how nested node params are translated in the latest i18n refactorings.